packages/core/useFavicon/index.ts

```diff
@@ -9,15 +10,20 @@ export interface UseFaviconOptions extends ConfigurableDocument {
   rel?: string
 }
 
+export type UseFaviconValue = string | null | undefined
+export type UseFaviconGetter = () => UseFaviconValue
+
 /**
  * Reactive favicon.
  *
  * @see https://vueuse.org/useFavicon
  * @param newIcon
  * @param options
  */
+export function useFavicon(newIcon: MaybeRef<UseFaviconValue>, options?: UseFaviconOptions): Ref<UseFaviconValue>
+export function useFavicon(newIcon: ComputedRef<UseFaviconValue> | UseFaviconGetter, options?: UseFaviconOptions): ComputedRef<UseFaviconValue>
```

Comment:
```suggestion
export function useFavicon(newIcon: MaybeComputedRef<UseFaviconValue>, options?: UseFaviconOptions): ComputedRef<UseFaviconValue>
```

Reply:
In this case the return type for Ref will be ComputedRef.
I mean for useFavicon(ref('str')): ComputedRef